yocto/fetch-sources-for-recipes.sh

```diff
@@ -27,6 +27,9 @@
 
 YOCTO_DIR="$1"
 IMAGES="$2"
+shift
+shift
```

Comment:
Could do `shift 2` here, I think.

Reply:
I think I prefer "shift 2", but if you want, we can leave it as is as well. :)

Reply:
[x] fixed